src/components/HomePageHeader/styles.module.css

```
@@ -9,11 +9,24 @@
 }
 
 .hero {
-  background: url('/static/img/bg-pattern-dark.png');
+  /* background: url('/static/img/bg-pattern-dark.png'); */
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: black;
+  height: 50vh;
```

more in line with the actual

```suggestion
  height: 32vh;
```